install-spacemacs/enhance-clojure-experience.md

```diff
@@ -12,9 +12,9 @@ Layers are defined in the `dotspacemacs-configuration-layers` section of the `~/
 | `clojure`         | [CIDER](https://docs.cider.mx) with [clj-refactor](https://github.com/clojure-emacs/clj-refactor.el/wiki) and [sayid](http://clojure-emacs.github.io/sayid/) debugger |
 | `git`             | git version control with [Magit](https://magit.vc/)                                                                                                            |
 | `markdown`        | writing project descriptions in README.md and other markdown files                                                                                                    |
-| `org-mode`        | write project documentaion, organise tasks                                                                                                                            |
+| `org-mode`        | write project documentation, organise tasks                                                                                                                            |
```

Comment:
Should this also be "`org`" rather than "`org-mode`"? (I see "`;; org`" in this section of my ~/.spacemacs and no "`org-mode`", but I'm new to this and can't immediately tell if "org" is something else.)

Reply:
Yes, it should be org.  I will check the usage and update all to org